### PR TITLE
Updated syntax

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Summary

Resolves issue #16662 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/document/anchors/index.md [hunk 1]
```diff
@@ -39,8 +39,9 @@ on the page:
 <title>Test</title>
 <script>
 function init() {
-  var toc = document.getElementById("toc");
-  var i, li, newAnchor;
+  const toc = document.getElementById("toc");
+  let i;
+  const li, newAnchor;
```
Comment (wbamberg) [inline files/en-us/web/api/document/anchors/index.md:44]: These need to be `let` because `const` must be initialized right away. Also while you are here we might prefer one line per declaration:

```suggestion
  let li;
  let newAnchor;
```

File: files/en-us/web/api/document/createtextnode/index.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ A {{domxref("Text")}} node.
 <title>createTextNode example</title>
 <script>
 function addTextNode(text) {
-  var newtext = document.createTextNode(text),
+  const newtext = document.createTextNode(text),
       p1 = document.getElementById("p1");
```
Comment (wbamberg) [inline files/en-us/web/api/document/createtextnode/index.md:44]:
```suggestion
  const newtext = document.createTextNode(text);
  const p1 = document.getElementById("p1");
```

File: files/en-us/web/api/document/domcontentloaded_event/index.md [hunk 3]
```diff
@@ -48,8 +48,8 @@ The following code ports the functionality of the `DOMContentLoaded` event all t
 
 function DOMContentLoaded() { "use strict";
 
-    var ael = 'addEventListener', rel = 'removeEventListener', aev = 'attachEvent', dev = 'detachEvent';
-    var alreadyRun = false,
+    const ael = 'addEventListener', rel = 'removeEventListener', aev = 'attachEvent', dev = 'detachEvent';
```
Comment (wbamberg) [inline files/en-us/web/api/document/domcontentloaded_event/index.md:51]: All the code you've edited in this file is in a polyfill, and we are not touching polyfills in this project, so please undo the changes for this file (domcontentloaded_event).

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @gullapalli9482 ! I had a few comments. Please be sure to ignore polyfills in this project.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for the updates, @gullapalli9482 . You need to undo _all_ the changes in https://github.com/mdn/content/pull/17071/files#diff-4702341d3a513054381ab78d6d2ab9ffae4c76e77c92180dc22b84474c5c06d8 .

(the domcontentloaded_event file)

File: files/en-us/web/api/document/domcontentloaded_event/index.md [hunk 4]
```diff
@@ -137,7 +137,7 @@ function DOMContentLoaded() { "use strict";
         // until we find `fn`, and then remove `fn` from its place in the array.
 
         if (typeof addOnload.queue === 'object') { // array
-            for (var i = addOnload.queue.length-1; i >= 0; i--) { // iterate backwards
+       for( var i = addOnload.queue.length-1; i >= 0; i--) { // iterate backwards
```
Comment (wbamberg) [inline files/en-us/web/api/document/domcontentloaded_event/index.md:140]:
```suggestion
             for (var i = addOnload.queue.length-1; i >= 0; i--) { // iterate backwards
```

File: files/en-us/web/api/document/domcontentloaded_event/index.md [hunk 5]
```diff
@@ -137,7 +137,7 @@ function DOMContentLoaded() { "use strict";
         // until we find `fn`, and then remove `fn` from its place in the array.
 
         if (typeof addOnload.queue === 'object') { // array
-            for (var i = addOnload.queue.length-1; i >= 0; i--) { // iterate backwards
+             for (var i = addOnload.queue.length-1; i >= 0; i--) { // iterate backwards
```
Comment (wbamberg) [inline files/en-us/web/api/document/domcontentloaded_event/index.md:140]:
```suggestion
            for (var i = addOnload.queue.length-1; i >= 0; i--) { // iterate backwards
```

**Comment by gullapalli9482** [comment]
Any other changes left to done?

**Review by wbamberg — APPROVED** [review]
OK, this looks good now. Thank you @gullapalli9482 !
